cash-register.js

```diff
@@ -6,102 +6,199 @@
 
```

Comment:
Okay done! :) So as a best practice, should I only keep code that should be reviewed up on Git? I shouldn't leave any 'copies' for reference for myself? I guess that's what local files are for right? Haha